pkg/destroy/aws/aws.go

```diff
@@ -1239,19 +1239,21 @@ func deleteRoute53(session *session.Session, arn arn.ARN, logger logrus.FieldLog
 	}
 
 	sharedEntries := map[string]*route53.ResourceRecordSet{}
-	err = client.ListResourceRecordSetsPages(
-		&route53.ListResourceRecordSetsInput{HostedZoneId: aws.String(sharedZoneID)},
-		func(results *route53.ListResourceRecordSetsOutput, lastPage bool) bool {
-			for _, recordSet := range results.ResourceRecordSets {
-				key := recordSetKey(recordSet)
-				sharedEntries[key] = recordSet
-			}
+	if len(sharedZoneID) != 0 {
```

Comment:
minor nit: It would be nice to have a debug log message when no shared zone is found.

Reply:
Thanks @staebler, nit addressed.